.goreleaser.yml

```diff
@@ -42,7 +42,7 @@ nfpms:
     bindir: /usr/local/sbin
     contents:
       - src: package-files/systemd/infra.service
-        dst: /usr/lib/systemd/system/infra.service
+        dst: /lib/systemd/system/infra.service
```

Comment:
The old path wasn't supported by systemd on debian (https://manpages.debian.org/bullseye/systemd/systemd.unit.5.en.html).

This new path seems to work on both.